### PR TITLE
Enable IT tests in phoenix-pig

### PR DESCRIPTION
**Pull request by gjacoby126**

### CODE REVIEW

File: phoenix-pig-base/pom.xml [hunk 1]
```diff
@@ -108,6 +108,18 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```
Comment (gjacoby126) [inline phoenix-pig-base/pom.xml:112]: Without this the IT tests were complaining they couldn't find the MiniDFSCluster class. Similar dependencies are in the phoenix-hive, etc projects, as is the disabling of the incorrect "dependency not used" error

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   5m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |  11m  2s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  16m 53s |  master passed  |
| +1 :green_heart: |  compile  |   1m 45s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 41s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |  16m 17s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 42s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 42s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 42s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   4m 44s |  phoenix-pig-base in the patch failed.  |
| +1 :green_heart: |  unit  |   2m 16s |  phoenix4-pig in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 18s |  phoenix5-pig in the patch passed.  |
| -1 :x: |  asflicense  |   0m 24s |  The patch generated 2 ASF License warnings.  |
|  |   |  65m 14s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix-connectors/pull/58 |
| Optional Tests | dupname asflicense javac javadoc unit xml compile |
| uname | Linux 44a28ccf937e 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-connectors-personality.sh |
| git revision | master / a732b66 |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/1/artifact/yetus-general-check/output/patch-unit-phoenix-pig-base.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/1/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/1/artifact/yetus-general-check/output/patch-asflicense-problems.txt |
| Max. process+thread count | 1508 (vs. ulimit of 30000) |
| modules | C: phoenix-pig-base phoenix-pig-base/phoenix4-pig phoenix-pig-base/phoenix5-pig U: phoenix-pig-base |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/1/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by stoty — APPROVED** [review]
+1 LGTM

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   7m 12s |  Docker mode activated.  |
| -1 :x: |  patch  |   0m  4s |  https://github.com/apache/phoenix-connectors/pull/58 does not apply to master. Rebase required? Wrong Branch? See https://yetus.apache.org/documentation/in-progress/precommit-patchnames for help.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix-connectors/pull/58 |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-58/2/console |
| versions | git=2.7.4 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**